datapoint/dptest/generator.go

```diff
@@ -79,3 +82,31 @@ func (d *EventSource) Next() *event.Event {
 func E() *event.Event {
 	return globalEventSource.Next()
 }
+
+// SpanSource is a simple way to generate throw away spans
+type SpanSource struct {
+	mu sync.Mutex
+
+	TimeSource func() int64
+	Name       string
+}
+
+var globalSpanSource SpanSource
+
+// Next return a unique span
+func (d *SpanSource) Next() *trace.Span {
+	d.mu.Lock()
+	defer d.mu.Unlock()
+
+	randomID := strconv.FormatInt(d.TimeSource(), 16)
+	return &trace.Span{
```

Comment:
Do you want to also generate a random start time (incremental) and duration (within a range)?

Reply:
Yeah, I can add that. I didn't need them for my testing, but I'm sure they will be useful for other testing later